### ISSUE TITLE
Encoding setting for Extra Courses results in sophomorix-check error

### ISSUE THREAD

**Issue by RolandBBerlin** [linuxmuster/sophomorix4#87]
In the WebUI settings it is possible to set encoding for Extra courses. Once this is done, however, adding new users is not possible, because ENCODING is not allowed for extraclasses in sophomorix-check.

![61773379-d5acae00-adf4-11e9-9d56-8441e2d5393a](https://user-images.githubusercontent.com/48863382/61775221-c6c7fa80-adf8-11e9-956c-fdf1326c46a5.png)

**Comment by PLanB2008** [comment]
duplicate 
Duplicate of #84

**Comment by jeffbeck** [comment]
sophomorix-extraclass is now a command. This avoids the encoding issue.
